proto/buf.gen.gogo.yaml

```diff
@@ -2,7 +2,7 @@ version: v1
 plugins:
   - name: gocosmos
     out: ..
-    opt: plugins=grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types
+    opt: plugins=grpc,Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types,Mcosmos/app/v1alpha1/module.proto=cosmossdk.io/api/cosmos/app/v1alpha1
```

Comment:
This is what enables the custom cosmos module options, right?

i.e.
```proto
option (cosmos.app.v1alpha1.module) = {
    go_import: "github.com/cosmos/ibc-go/modules/capability"
};
```

Reply:
yeah, that's my understanding.